HOSE.md

```diff
@@ -1,6 +1,47 @@
 # Pack Hose Replacements
 
-## Hose Example #1
+In order to deliver both power and data signals to the wand, a new set of cabling is required. As wiring is typically desired to remain internal to the hose the following approaches have been used to replace the stock 2-wire (power-only) hose with a new 4-wire solution. The existing wire loom (black hose) can be reused if desired, or replaced with a more flexible brand.
+
+## PCB Kit Hose
+
+For those who have purchased the PCB kits with the hose replacement, the following parts are utilized:
+
+* 1 - [CNLINKO 4-Pin Connector](https://a.co/d/8ALa9Gq) - Connector for the pack which uses a square connector roughly the size of the stock connection
+* 1 - [Neutrik NL4FX 4-Pole Locking SpeakON Cable Connector](https://a.co/d/9c7djjo) - This is the connector at the wand end of the hose
+* 1 - [Neutrik NL4MP 4-Pole SpeakON Panel Mount](https://a.co/d/atMuNfz) - This is the socket for the wand itself
```

Comment:
The NL4MP-ST is the one we are going to be providing. It has screw terminals and requires no soldering.

https://www.neutrik.com/en/product/nl4mp-st